hbw/app/javascript/packs/hbw/components/claiming/task_list.jsx

```diff
@@ -53,13 +53,15 @@ class HBWClaimingTaskList extends Component {
   claim = async (task) => {
     const { activeTask } = this.state;
     const { connection } = this.props.env;
+    const { refresh } = this.context;
 
     this.setState({ claimingTask: task });
 
     await connection.request({
       url:    `${connection.serverURL}/tasks/${task.id}/claim`,
       method: 'POST'
-    });
+    })
+      .done(() => refresh(task.id));
```

Comment:
Ether `async/await` or Promise callbacks, don't mix it please. 
We prefer modern `async/await` syntax 😎 .